packages/database/src/migrations/20210128102520-PalauFacilityOpeningHoursDashboard-modifies-data.js

```diff
@@ -68,12 +68,42 @@ const dashboardReport = {
   },
 };
 
+const elementList = [
+  'FF20',
+  'FF21',
+  'FF22',
+  'FF23',
+  'BCD39D',
+  'FF24',
+  'FF25',
+  'BCD39E',
+  'FF26',
+  'FF27',
+  'BCD39F',
+  'FF28',
+  'FF29',
+  'BCD39G',
+  'FF30',
+  'FF31',
+  'BCD39H',
+  'FF32',
+  'FF33',
```

Comment:
Can you confirm all of these have been checked to make sure:
- They don't exist on DHIS2
- No visualisation currently relies on them (or if one or more do, they will be added to regression testing notes)
- They don't relate to questions asking about medicine availability

Reply:
- They don't exist on DHIS2 ❌ (BCD39D - DCD39H are on DHIS2, others don't)
- No visualisation currently relies on them (or if one or more do, they will be added to regression testing notes) ✅ Checked
- They don't relate to questions asking about medicine availability ✅ The answer are open and close time on each day 

My bad, I should be more careful on the first question. 
Additional info: `BCD39D - DCD39H` indicates whether it is open or not per day, the rest are open and close time on each day.